units/src/lib.rs

```diff
@@ -27,10 +27,6 @@ extern crate alloc;
 #[cfg(feature = "std")]
 extern crate std;
 
-/// A generic serialization/deserialization framework.
-#[cfg(feature = "serde")]
-pub extern crate serde;
```

Comment:
Originally I was thinking of reexporting it in `units` and not reexporting it in `primitives` (but using globs would inherit it after all). But this is not really wrong either.